src/components/SortDropdown/SortDropdown.tsx

```diff
@@ -22,32 +21,41 @@ export interface SortDropdownProps {
   mobile?: boolean;
 }
 
+export const toggleFilters = (event: Omit<MouseEvent, "currentTarget"> & {
```

Comment:
I understand this is toggling the filters & sort modal. I would move it to a different file, and not make it part of the SortDropdown